esp/src/eclwatch/LZBrowseWidget.js

```diff
@@ -676,7 +676,9 @@ define([
                                 if (!fsize || fsize === -1) {
                                     return ""
                                 }
-                                return fsize.toLocaleString();
+                                var unitConversion = ["Bytes","KB","MB","GB","TB","PB","EB","ZB","YB"],
```

Comment:
No 'Zero bytes' here?

Reply:
This is in the Landing zone, prior to conversion of unit size, we did not put a size measurement in the dropzone / node ip/ directory level, those return blank as prior. 

Not sure if we want that to change, or what should be written there.